### ISSUE TITLE
Destroy lock_manager without unlocking

### ISSUE THREAD

**Issue by scott-8** [joanvila/aioredlock#103]
One feature that would be very helpful is the ability to destroy a lock manager without unlocking any locks (and allowing them to expire at their given timeout).

My current plan is to keep it simple and add an boolean attribute to `Lock` named something like `unlock_on_exit`, and check it before calling `self.redis.unset_lock`.

I'm willing to make a PR for this, and want to hear your thoughts before moving forward.

**Comment by gtmanfred** [comment]
yeah, that sounds like a great idea. especially if it does not change the default behavior.
